platform/spire/src/virt.py

```diff
@@ -19,18 +19,18 @@
 
 
 def get_bridge(ip):
-    return "spirebr%s" % hex(ip.to_integer())[2:].upper()
+    return "spirebr%s" % ip.packed.hex().upper()
 
 
 def get_node_tap(node):
     # maximum length: 15 characters
-    return "spirtap%s" % hex(node.ip.to_integer())[2:].upper()
+    return "spirtap%s" % node.ip.packed.hex().upper()
```

Comment:
Did we determine that this was going to change the format slightly, due to padding? I seem to remember we hit some weird issues around `spire virt net down` as a result of this, so it might be worth calling out in the commit message.